src/modal.css

```
@@ -0,0 +1,114 @@
+.filters-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  button {
+    display: inline-block;
+    background-color: #262626;
+    color: white;
+    width: fit-content;
+    height: 42px;
+    padding: 8px 16px;
+    border-radius: 21px;
+    font-size: 19px;
+    line-height: 19px;
+    font-weight: 700;
+    border: 1px solid #262626;
+  }
+  .disabled-filter {
+    background-color: white;
+    color: #262626;
+    border: 1px solid #dbd9d7;
+  }
+}
+
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.67);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-content {
+  background: white;
+  padding: 40px 24px;
+  border-radius: 8px;
+  width: 496px;
+  text-align: center;
+  margin: 16px;
+}
+
+.search-bar input[type="text"] {
```

To be consistent with the nesting approach at the top this should be nested as well.

```suggestion
.search-bar {
  input[type="text"] {
```